requirements.md

```diff
@@ -1,55 +1,52 @@
 # ESGF-1.5 Requirements and Acceptance Criteria
 
-This document outlines the requirements and acceptance criteria for the ESGF2-US's ESGF-1.5 deployment. ESGF-1.5 is a new version of the US-Dept. of Energy's (US-DOE) ESGF catalog and data archive for CMIP6 and prior ESGF data. (CMIP7 and beyond will instead use the ESGF-NG system.) ESGF-1.5 will allow US-DOE to continue providing access to legacy ESGF data by migrating to new, more easily-maintainable systems.
+This document outlines the requirements and acceptance criteria for the ESGF2-US ESGF-1.5 deployment. ESGF-1.5 is a new version of the US-Dept. of Energy's (US-DOE) ESGF catalog and data archive for CMIP6 and prior ESGF data. (CMIP7 and beyond will instead use the ESGF-NG system.) ESGF-1.5 will allow US-DOE to continue providing access to legacy ESGF data by migrating to new, more easily-maintainable systems. Given that the ESGF-1.5 deployment targets legacy ESGF data, we will minimize the effort and cost to migrate from ESGF-1.0 by making the minimum necessary changes to maintain legacy data access. Ideas for improvements should target ESGF-NG.
 
 ## Changes from ESGF-1.0
 
-- **Discovery** - Replace Apache Solr search indices with Globus Search indices.
-- **Discovery** - (Eventually) Consolidate the separate indices at Lawrence Livermore, Argonne, and Oak Ridge national laboratories into a single US-DOE index.
+- **Discovery** - Replace Apache Solr search indices with a single Globus Search index
 - **Data Access** - Replace legacy HTTP/S servers (for data access) with Globus Connect Server (providing both HTTP/S and bulk data transfer)
 - **Data Access** - Update wget script service to use Globus HTTP/S and begin a deprecation process
 - **Applications** - Replace COG web application with MetaGrid
 - **Applications** - Add intake-esgf command-line interface
 
-## Globus Search indices
+## Globus Search Index
 
-**Phase I:** Initially, each US-DOE laboratory (LLNL, ANL, ORNL) will replace its Solr ESGF index with a distinct Globus Search ESGF index.
-
-**Phase II:** As a separate follow-on activity, the three laboratories will consolidate the three indices into a single US-DOE index that references data wherever it is available.
+Replace Apache Solr search indices at Lawrence Livermore, Argonne, and Oak Ridge National Laboratories with a single consolidated US-DOE Globus Search index referencing all available data replicas.
 
 ### Index contents
 
-In Phase I, each index MUST contain metadata for data stored at the corresponding site (ANL, LLNL, ORNL).
-
-In Phase II, the consolidated index MUST contain metadata for data stored at ANY US-DOE site (ANL, LLNL, ORNL).
+Each dataset will have a single entry in the consolidated index, which will refer to ALL US-DOE locations where the data may be accessed.
```

Comment:
This is not how I understand things, but I could be confused. I understand that every *dataset* in ESGF has a record in the index *per location* it is stored. See this [search](https://esgf-node.llnl.gov/esg-search/search?instance_id=CMIP6.CMIP.NCAR.CESM2.historical.r1i1p1f1.Lmon.gpp.gn.v20190308&distrib=0&format=application%2Fsolr%2Bjson) where I search for a particular dataset. With distributed search turned off, we find 2 dataset records for the same data at different locations. Have I missed something @sashakames ? I think that this means that per dataset we will have:

1. An index record *per location* that a dataset is being stored.
2. Each dataset record will have possibly many file index records.

Reply:
There is one Type=Dataset index entry, and one or more related Type=File index entries representing file replicas.